tests/EmbeddedRelationsTest.php

```diff
@@ -945,4 +946,13 @@ public function testGetQueueableRelationsEmbedsOne()
         $this->assertEquals(['father'], $user->getQueueableRelations());
         $this->assertEquals([], $user->father->getQueueableRelations());
     }
+
+    public function testEmbedManySerialization()
+    {
+        $user = User::create(['name' => 'John Doe']);
+        $user->addresses()->save(new Address(['city' => 'New York']));
+
+        $results = $user->toArray();
+        assertIsString($results['addresses'][0]['_id']);
```

Comment:
We use the `TestCase` method.
```suggestion
        $this->assertIsString($results['addresses'][0]['_id']);
```